configuration-file-v2.rst

```diff
@@ -0,0 +1,127 @@
+.. post:: January 30, 2019
+   :tags: feature
+   :author: Santos
+   :location: CUE
+
+New Configuration File
+======================
+
+We are happy to announce the new version of the `Read the Docs configuration file`_ (v2).
+
+.. _Read the Docs configuration file: https://docs.readthedocs.org/en/latest/config-file/v2
+
+.. code-block:: yaml
+
+   version: 2
+   python:
+     version: 3.7
+     install:
+       - requirements: docs/requirements.txt
+   sphinx:
+     configuration: docs/conf.py
+   formats: all
+
+If you are a recurrent Read the Docs user,
+chances are that you've configured your projects using a `.readthedocs.yml` file.
+
+Using the web interface is quick,
+but the main advantages of using a configuration file over the web interface are:
+
+- Some settings are only available using a configuration file.
+
+We are moving away from using the web interface.
+
+- The settings are per version rather than per project.
+   
+This would allow different versions of docs to be built with different versions of Python
+or different requirements.
+
+- The settings live in your VCS.
+
+- Reproducible build environments over time.
+
+What did we change in the new version?
+-------------------------------------
+
+- All valid options from the web interface are available.
```

Comment:
This should probably have some introduction that isn't in bullet form. Having an entire section that is just bullets feels overly brief.